apps/relayer/tests/constants.ts

```diff
@@ -9,7 +9,7 @@ export const INT_STATE_TREE_DEPTH = 1;
 export const VOTE_OPTION_TREE_DEPTH = 2;
 export const MESSAGE_BATCH_SIZE = 20;
 export const DEFAULT_INITIAL_VOICE_CREDITS = 99;
-export const DEFAULT_VOTE_OPTIONS = 25;
+export const DEFAULT_VOTE_OPTIONS = 20;
```

Comment:
not really important but why 20? default should be 5 ** voteOptionTreeDepth so 25

Reply:
There is a validation check that is copied from cli:

```
  // ensure the vote option parameter is valid (if passed)
  if (voteOptions && voteOptions >= VOTE_OPTION_TREE_ARITY ** voteOptionTreeDepth) {
    throw new Error("Vote options cannot be greater than the number of leaves in the vote option tree");
  }
```

It allows only 24 options in this case. I can change it to `>` to allow 25.